examples/src/main/java/org/apache/spark/examples/ml/JavaBucketedRandomProjectionLSHExample.java

```diff
@@ -71,25 +71,32 @@ public static void main(String[] args) {
     BucketedRandomProjectionLSH mh = new BucketedRandomProjectionLSH()
       .setBucketLength(2.0)
       .setNumHashTables(3)
-      .setInputCol("keys")
-      .setOutputCol("values");
+      .setInputCol("features")
+      .setOutputCol("hashes");
 
     BucketedRandomProjectionLSHModel model = mh.fit(dfA);
 
     // Feature Transformation
+    System.out.println("The hashed dataset where hashed values are stored in the column 'values':");
     model.transform(dfA).show();
     // Cache the transformed columns
     Dataset<Row> transformedA = model.transform(dfA).cache();
     Dataset<Row> transformedB = model.transform(dfB).cache();
 
     // Approximate similarity join
+    System.out.println("Approximately joining dfA and dfB on distance smaller than 1.5:");
     model.approxSimilarityJoin(dfA, dfB, 1.5).show();
+    System.out.println("Joining cached datasets to avoid recomputing the hash values:");
     model.approxSimilarityJoin(transformedA, transformedB, 1.5).show();
+
     // Self Join
+    System.out.println("Approximately self join of dfB on distance smaller than 2.5:");
```

Comment:
I don't think the self join is necessary, but I'll defer to others. Also, "Approximate self join" and it's dfA not dfB